pwnlib/fmtstr.py

```diff
@@ -697,6 +698,13 @@ def make_payload_dollar(data_offset, atoms, numbwritten=0, countersize=4):
     fmt = ""
 
     counter = numbwritten
+
+    if no_dollars:
+        # since we cant dynamically offset, we have to increment manually the parameter index, use %c, so the numeber of bytes written is predictable
+        fmt += "%c" * ( data_offset - 1)
```

Comment:
A couple typos / formatting issues

Reply:
do i have to fix something about it?

Reply:
numeber, cant≠can't, space after (

Reply:
are you talking bout the space missing at the end of the  parenthesis?

Reply:
ok corrected them, thx , i ll close this one.